src/components/common/Loader.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <a-spin :class="loaderClassName" tip="Загрузка..."></a-spin>
+</template>
+
+<script>
+export default {
+  name: 'Loader',
+  props: {
+    className: String,
+  },
+  computed: {
+    loaderClassName() {
+      return this.className || 'loader';
+    },
+  },
+};
+</script>
+
+<style lang="scss">
+.loader {
+  margin-top: 40vh !important;
+
+  &__wrapper {
```

Comment:
It doesn't look like you are using this style

Reply:
I use it in another file (App.vue), it is default style, but if I don't need it - I can insert loader and write for this another wrapper styles (where is no need to show loader on full page with blur, for example)